code/modules/mob/language.dm

```diff
@@ -207,6 +207,15 @@
","Acantha","Addilyn","Adela","Adele","Aderyn","Adley","Adriana","Aerona","Aileen","Alanis","Alberta","Alex","Alexandra","Alice","Alma","Amalie","Andromeda","Angel","Ann","Anna","Anne","Annie","Ariel","Arya","Astraea","Astrid","Athena","Audra","Aura","Aurora","Avice","Bailey","Barbara","Beatrix","Belinda","Bellatrix","Belle","Bianca","Braelynn","Brea","Bree","Brooke","Brunhilde","Caitlin","Callisto","Camden","Camilla","Candra","Carina","Carletta","Carline","Carman","Caroline","Carys","Cassandra","Cassiopeia","Catlyn","Celeste","Celina","Ceres","Cerian","Charlotte","Chelle","Chloe","Cicely","Clara","Clarice","Claudia","Cordelia","Cornelia","Cressida","Cybele","Dagmar","Daisy","Dakota","Daphne","Daria","Darla","Dawn","Deidra","Deja","Delilah","Delphine","Delyth","Demetra","Dezra","Dinah","Dora","Effie","Eiddwen","Elaine","Elara","Eleanor","Eliana","Elise","Ellen","Elsa","Emilia","Emily","Emma","Emmie","Emmy","Ester","Esther","Eva","Fadila","Farren","Fay","Felita","Felizia","Fenella","Fleur","Francesca","Freya","Garnet","Ginger","Grace","Gracie","Gwen","Hadley","Hafren","Halley","Hannah","Harlyn","Harmony","Harper","Hazel","Helen","Helena","Helene","Hilda","Holly","Honey","Hope","Idonea","Igna","Imogen","Ina","Iona","Irene","Irma","Isabel","Isabella","Ivy","Jacqueline","Jaelyn","Jana","Janice","Janis","Jayene","Jazzlyn","Jeane","Jennete","Jennifer","Jill","Jo","Johanna","Joslyn","Juliana","Juliet","June","Kaia","Kali","Karlene","Kathryn","Kenna","Kiera","Kiley","Kimberly","Kivela","Lacey","Lachelle","Lacy","Larissa","Laura","Layla","Lena","Leonor","Leslie","Lexi","Liana","Liani","Lianne","Liesel","Lili","Liliwen","Lilly","Linda","Lola","Lona","Lorelai","Lorelei","Luise","Lulu","Luna","Lycia","Lyn","Mabyn","Madeleine","Maeve","Magdalene","Maggie","Maia","Maragaret","Margarethe","Mariah","Mariam","Marilyn","Marina","Marisole","Marivel","Marley","Marni","Marrie","Martina","Mary","Mazelina","Meda","Medea","Mei","Melania","Melanie","Melody","Mercedes","Merle","Meryl","Mia","Michelle","Mila","Millie","Mindy","Miranda","Missy","Misty","Mona","Morgan","Morgana","Morrigan","Morticia","Nadia","Nadine","Nessa","Nia","Nicole","Nikki","Nimah","Nina","Norma","Nova","Olive","Olivia","Opaline","Ophelia","Oriana","Paisley","Paloma","Pam","Pauline","Paz","Penelope","Penny","Phoebe","Piper","Portia","Priya","Rachel","Raina","Raura","Raven","Rayna","Rayne","Rebecca","Regina","Renee","Rhea","Rina","Robin","Rosalind","Rosie","Rowen","Rowena","Royce","Rubella","Ruby","Rue","Ruth","Sabrina","Sadie","Sahara","Sandra","Savina","Sawyer","Selene","Sena","Seraphina","Seraphine","Sheba","Sheila","Sia","Sibylle","Sofie","Sonnet","Sophia","Stacia","Stacy","Stefanie","Stella","Suri","Syden","Sylvia","Tala","Tasha","Tasia","Tatum","Taylee","Teegan","Teresa","Tess","Tessa","Tessy","Theia","Titania","Trisha","Trixie","Trudy","Uma","Ursula","Valenia","Valentina","Vega","Vera","Verena","Vicky","Victoria","Willow","Winnie","Yasmin","Yvette","Yvonne","Zia","Zinnia","Ziva","Zoe","Zuri"))]"
```

Comment:
Yeah these are gonna need to be a config.

Take a look at how other names do it with `code\_globalvars\lists\names.dm`